docs/pages/technical-specs/expo-updates-1.mdx

```diff
@@ -60,47 +68,81 @@ A conformant client library configured to perform [code signing](#code-signing)
 Example:
 
 ```
+expo-protocol-version: 1
 accept: application/expo+json;q=0.9, application/json;q=0.8, multipart/mixed
 expo-platform: *
 expo-runtime-version: *
 expo-expect-signature: sig, keyid="root", alg="rsa-v1_5-sha256"
 ```
 
-## Manifest Response
+## Response
 
-A conformant server MUST return a response structured in at least one of two ways. A conformant server MAY support either or both response structures, and when an unsupported response structure is requested the server SHOULD respond with an HTTP `406` error status.
+A conformant server MUST return a response structured in at least one of the two following response structures, MAY support either or both response structures, and when an unsupported response structure is requested the server SHOULD respond with an HTTP `415` error status. A server that wishes to respond with an incompatible response for the requested protocol version SHOULD respond with an HTTP `404` error status instead.
```

Comment:
I went with 406 and 406 here since returning a 200 in this case might mean we'd want to return it in all cases and that seems like it might be a bigger change.